pyfftw/interfaces/scipy_fft.py

```diff
@@ -370,8 +368,10 @@ def dct(x, type=2, n=None, axis=-1, norm=None, overwrite_x=False,
 
     """
     threads = _workers_to_threads(workers)
+    # not sure why's that if-clause necessary, but some `scipy_fft`
+    # tests fail otherwise
     if norm is None:
-        norm = 'backward'
+        norm = "backward"
```

Comment:
also not sure why that change is necessary in the `scipy_fft` sine/cosine transforms

Reply:
This seems suspicious to me...

Reply:
both this if-clause and the mapping `None:None` referenced in my comment above seem to be related to the work done on #256.

I don't want to start changing too many things unrelated to the current PR, so I left them unchanged to make sure no tests are failing. Could be looked into with a new PR that focuses on changes leftover from #256 and similar work. 

@grlee77

Reply:
Ok, but changing single quotes to double quotes shouldn't change anything.

Reply:
no the single-to-double quotes change was just done while I was doing aesthetic changes in bulk (unrelated); the change I'm referring to is why should one need to change `norm=None` to `norm="backward"` for sine/cosine transforms, even though they're defined as equivalent parameters.

Such a change is not necessary for any of the other FFTs.

Reply:
Yeah, not sure about that.

As an aside, it would be appreciated if you didn't make bulk minor stylistic changes - it makes parsing the PR much harder and is arguably undesirable. Changes to be PEP8 compliant are valued, but just changing single quotes to double quotes seems like unnecessary noise to me.

Reply:
you're right I'm sorry about that; I made them in bulk through my editor so I underestimated how noisy it makes the PR changes look

Reply:
There's too much noise that is impossible for non-maintainers like me to help review (not saying I have time right now, but still)...(btw I made the counterpart change in CuPy: cupy/cupy#4797, cupy/cupy#4816) It'd be really nice if this PR can be split into two.